pkg/machine/e2e/init_test.go

```diff
@@ -308,6 +305,29 @@ var _ = Describe("podman machine init", func() {
 		Expect(sshSession.outputToString()).To(ContainSubstring("example"))
 	})
 
+	It("machine init with volume with spaces", func() {
+		skipIfWSL("WSL volumes are much different.  This test will not pass as is")
+
+		tmpDir, err := os.MkdirTemp("", "with spaces")
+		Expect(err).ToNot(HaveOccurred())
+		_, err = os.CreateTemp(tmpDir, "example")
+		Expect(err).ToNot(HaveOccurred())
+		mount := tmpDir + ":/host folder"
+		defer func() { _ = utils.GuardedRemoveAll(tmpDir) }()
+
+		name := randomString()
+		i := new(initMachine)
+		session, err := mb.setName(name).setCmd(i.withImage(mb.imagePath).withVolume(mount).withNow()).run()
```

Comment:
please merge this with `machine init with volume`

echo machine init+boot takes around 20s or more in CI so this slows down the tests a lot. We need to avoid spawning so many VMs if we want to keep the performance of the suite reasonable.

I would just add two volumes in the test above.

Reply:
This comment is still open, we really need to reduce the number of machine init/starts in the suite to keep the test time sane

Reply:
Thanks. This should be addressed now.